cmake/doc.cmake

```diff
@@ -1,22 +1,9 @@
-find_program(PANDOC_EXECUTABLE pandoc)
-if(NOT PANDOC_EXECUTABLE)
-        message(WARNING "Pandoc not found, will not generate manpages")
-        return()
-endif()
-
 string(TIMESTAMP TODAY "%B %d, %Y")
 
-add_custom_command(
```

Comment:
Presumably at some point in the future you'll want to update the documentation. It may be  a good idea to keep the manpage generation commands here, at least to facilitate manual execution when that time comes up.  It'll ensure that you're executing pandoc with the same arguments, etc.

Beyond that, this is all good with me.

Reply:
good point.  I made it optional by passing -DGENERATE_MANPAGES so anyone can opt into it and it's trivial to regenerate when updating.  thanks!